src/main/java/core/basesyntax/service/interfaces/FruitFileSaver.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.interfaces;
+
+public interface FruitFileSaver {
+    void saveToFile(String report, String filePath);
```

Comment:
There is no need to be specific on what you save here, because this interface should be reusable in other parts of the program and not just fruits
```suggestion
public interface FileWriter {
    void writeDataToFile(String data, String filePath);
```